packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -1,25 +1,35 @@
 import {
 	Component,
+	ElementRef,
 	EventEmitter,
 	Inject,
 	Input,
 	OnChanges,
+	OnDestroy,
 	OnInit,
 	Output,
+	Renderer2,
 	SimpleChanges,
-	ViewEncapsulation,
-	ElementRef,
-	Renderer2
+	ViewEncapsulation
 } from "@angular/core";
 import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
 import { AbstractStarkUiComponent } from "../../../common/classes/abstract-component";
-import { StarkComponentUtil } from "../../../util/component";
+import { StarkComponentUtil, StarkFormUtil } from "../../../util";
+import { FormControl, Validators } from "@angular/forms";
+import { distinctUntilChanged, filter } from "rxjs/operators";
+import { Subscription } from "rxjs";
 
 /**
  * Name of the component
  */
 const componentName: string = "stark-dropdown";
 
+/**
+ * Warning message for reactive form issue
+ */
+const reactiveWarningMessage: string =
+	componentName + ': When dropdownFormControl is set, "value", "selectionChanged" and "isDisabled"should be undefined.';
```

Comment:
Missing whitespace... and small change: "When dropdownFormControl is set, "value", "selectionChanged" and "isDisabled" should not be defined."